tests/Parse/MarkdownTest.php

```diff
@@ -75,7 +75,7 @@ public function testFrontMatter()
      *
      * @return array
      */
-    public function markdownData()
+    public static function markdownData()
```

Comment:
What was the reason that required this to be changed to static?

Reply:
I think I changed this to support PHPUnit 10. I later decided to not add that support, as Laravel 10 is big enough as an update. I reverted these changes.